src/services/api-exem.ts

```diff
@@ -0,0 +1,109 @@
+import axios, { type Method } from 'axios'
+
+export async function getSiteMeasurement (id: string) {
+
+  const baseUrl = 'https://external-api.exem.fr/sites'
+  const api_key = 'AIzaSyCCq6wUPLEBATfwUbcRU9bCsGmnp0IOJCU'
```

Comment:
@anaislet-c2c I think we shouldn't put the api_key in the code but we need to use the env variable or it won't fulfill the purpose of having a backend the first time

cc @MaYohann